engine/src/main/java/org/terasology/config/PermissionConfig.java

```diff
@@ -13,28 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.terasology.config;
 
 import java.security.SecureRandom;
 import java.util.ArrayList;
 import java.util.List;
 
-/**
- */
+
 public class PermissionConfig {
```

Comment:
Public classes should always have a javadoc. If you cannot write one feel free to leave:

```
/**
* TODO
*/
```

so that somebody at some point will notice it and write it.